docs/src/assets/scss/languages.scss

```diff
@@ -13,10 +13,10 @@
 
     a {
         color: inherit;
-        display: block;
         width: 100%;
         padding: .75rem .1rem;
         text-decoration: none;
+        display: block;
         display: flex;
```

Comment:
interesting 😂  we have both block and flex

Reply:
Is it possible for us to check which value takes precedence and gets applied?

Reply:
I assumed it might have been another Old IE thing, so I left it on the second pass.

Reply:
I think as a separate discussion, something like PostCSS might be able to calculate those variables as a build step based on a supported browser range. It's been awhile since I looked at that type of tooling though